docs/root/version_history/current.rst

```diff
@@ -64,6 +64,7 @@ Removed Config or Runtime
 
 New Features
 ------------
+* compression: added new runtime guard `envoy.reloadable_features.enable_compression_without_chunked_header` (enabled by default) to control compression if no transfer-encoding=chunked header exists (now envoy would compress in such cases) to account for http/2 connections, where such transfer-encoding were removed.
```

Comment:
compression: extended the compression allow compressing when the content length header is not present.  This behavior may be temporarily reverted by setting `envoy.reloadable_features.enable_compression_without_chunked_header` to false.